stingray/crossspectrum.py

```diff
@@ -0,0 +1,380 @@
+__all__ = ["Crossspectrum", "AveragedCrossspectrum"]
+
+import numpy as np
+import scipy
+import scipy.stats
+import scipy.fftpack
+import scipy.optimize
+
+import stingray.lightcurve as lightcurve
+import stingray.utils as utils
+
+
+class Crossspectrum(object):
+
+    def __init__(self, lc_1=None, lc_2=None, norm='none'):
+        """
```

Comment:
I am wondering whether it would make sense to refactor `lc_1` and `lc_2` to read `lc1` and `lc2`? Same with `nphots_1` and `nphots_2` below. The additional underscores are just *slightly* more work and would probably annoy me if I had to use the class a lot in practice.